### DISCUSSION TITLE
Don't show keypresses on keyboard when typing hidden input in terminal

### DISCUSSION THREAD

**Discussion by danbulant** [GitSquared/edex-ui#1035] [Ideas]
I'm not sure how much technically possible is this, but if you could check the input type in terminal (xterm should have some kind of cache - maybe check for some input styles or something?) and hide the keypresses when the input is not visible, it could allow recording videos when typing passwords (e.g. in sudo, it hides the password input).

**Comment by GitSquared** [comment]
Check out this [thread](https://github.com/GitSquared/edex-ui/issues/394#issuecomment-484571538) (2019). Password mode (ctrl+shift+p) was implemented to allow disabling visual input feedback, but there is really no clean, cross-platform way to detect "hidden" inputs automatically.
